api-core/src/main/java/com/vimeo/networking2/VimeoCallback.kt

```diff
@@ -42,3 +44,17 @@ interface VimeoCallback<ResponseType_T> {
     fun onError(error: VimeoResponse.Error)
 
 }
+
+/**
+ * Create a lambda friendly instance of the [VimeoCallback].
```

Comment:
Are you suggesting remove "Create?" The usage of create is accurate, as it creates an instance of a callback.

Reply:
Sorry. I had a brain fart writing that. I meant to write "Creates a lambda friendly..." obviously a completely trivial and optional remark. I just thought we tend to write these from the perspective of what the function does rather than what the caller can do with the function.

Feel free to ignore it.